ansible/roles/ansible-tvault-contego-extension/tasks/install_rhel.yml

```diff
@@ -16,6 +16,19 @@
 - name: Reload Daemon
   shell: systemctl daemon-reload
 
+- name: Check if centos-release-openstack-{{OPENSTACK_DIST}} is installed
+  command: rpm -q centos-release-openstack-{{OPENSTACK_DIST}}
+  register: rpm_check
+  ignore_errors: yes
```

Comment:
we can ignore this rpm check, instead we can check the return code of the yum update_cache task.